apps/rxjs/14-race-condition/src/app/app.component.ts

```diff
@@ -1,39 +1,29 @@
-import {
-  ChangeDetectionStrategy,
-  Component,
-  inject,
-  OnInit,
-} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
 import { MatDialog } from '@angular/material/dialog';
-import { take } from 'rxjs';
 import { TopicModalComponent } from './topic-dialog.component';
 import { TopicService, TopicType } from './topic.service';
 
 @Component({
   standalone: true,
   selector: 'app-root',
+  imports: [CommonModule],
   template: `
-    <button (click)="openTopicModal()">Open Topic</button>
+    <ng-container *ngIf="topics$ | async as topics">
```

Comment:
try to always refactor to the new @if synthax. You should try to get use to it. That's the future

Same for signal, don't use async pipe anymore, convert to `toSignal` so you will benefit from the new signal component when they get release. And from the hybrid change detection in v18